pkg/jiraservicedesk/client/client_test.go

```diff
@@ -0,0 +1,78 @@
+package client
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/nbio/st"
+	"gopkg.in/h2non/gock.v1"
+)
+
+var endpoint = "177"
+var client = NewClient("", "https://mock.atlassian.net/", "")
```

Comment:
Don't initialize client here. Either do it in the test case or add a `beforeAll` method if it's supported.